Gems/ScriptCanvas/Code/Include/ScriptCanvas/Execution/Interpreted/ExecutionStateInterpretedPerActivation.h

```diff
@@ -32,7 +30,20 @@ namespace ScriptCanvas
         void StopExecution() override;
 
     protected:
-        bool m_deactivationRequired = false;
+        bool m_deactivationRequired;
+
+        void ClearLuaRegistryIndex();
+
+        int GetLuaRegistryIndex() const;
```

Comment:
Moved from the base class to this one, since no pure functions need this functionality.